static/app/views/starfish/components/chart.tsx

```diff
@@ -370,6 +407,16 @@ function Chart({
                         animation: false,
                       })
                   ),
+                  ...incompleteSeries.map(({seriesName, data: seriesData, ...options}) =>
+                    LineSeries({
+                      ...options,
+                      name: seriesName,
+                      data: seriesData?.map(({value, name}) => [name, value]),
+                      animation: false,
+                      animationThreshold: 1,
+                      animationDuration: 0,
+                    })
+                  ),
```

Comment:
Should this be placed between `series` and `scatterPlot`? IIRC, echarts stack the charts on top of one another and this can place the incomplete series on top of the scatter plot.

Reply:
Yeah this will have the side effect of overlapping on top of any scatter plots. But IMO the effect is barely noticeable:
<img width="151" alt="image" src="https://github.com/getsentry/sentry/assets/83961295/b94d5636-a240-4e7b-856f-3819e1027be3">

I opted to put the series at the end to minimize code change, because we have some charts + hover handlers that always assume the scatter plot comes after the initial line series.

lmk if you have a strong opinion on this though